test/library/packages/LinearAlgebra/correctness/testSVDNative.chpl

```diff
@@ -0,0 +1,26 @@
+use LinearAlgebra;
```

Comment:
Good initial tests. I think it would be beneficial to cover some of the edge cases (over/under-determined matrices, singular matrices).

The [`scipy.linalg`](https://github.com/scipy/scipy/blob/master/scipy/linalg/tests/test_decomp.py#L1041-L1086) has some example cases to check against.

If we end up not adding complex support for this PR, we ought to add a future for it (let me know if you have any questions about that process).

Reply:
Also, a performance test would be great, since we intend to optimize this implementation going forward.

In fact, getting LAPACK svd vs. Chapel svd on the same nightly performance-tracking plot could be really cool

Reply:
I'll add those ASAP